doc/source/whatsnew/v0.20.0.txt

```diff
@@ -580,3 +580,5 @@ Bug Fixes
 - Bug in ``Series.replace`` and ``DataFrame.replace`` which failed on empty replacement dicts (:issue:`15289`)
 - Bug in ``pd.melt()`` where passing a tuple value for ``value_vars`` caused a ``TypeError`` (:issue:`15348`)
 - Bug in ``.eval()`` which caused multiline evals to fail with local variables not on the first line (:issue:`15342`)
```

Comment:
just an FYI, if you put changes in whatsnew NOT at the end (but in empty space that is left), then you won't get conflicts.